docs/create_equinix_cluster.md

```diff
@@ -0,0 +1,109 @@
+# Create Equinix Metal cluster using CAPI / CAPEM
+
+## Overview
+
+- We are using machines contributed to the CNCF by Equinix Metal.
+- These steps manually create a cluster using CAPI / CAPEM using a local
+Kind cluster for the management cluster.
+- These docs are based on the Equinix CAPI [guide](https://deploy.equinix.com/developers/guides/kubernetes-cluster-api/).
+- Using [Podman](https://github.com/kubernetes-sigs/cluster-api-provider-packet/issues/496)
+is recommended by Equinix for MacOS.
+- We plan to automate these steps later using GitHub Actions and an IaC tool.
+
+## Create Cluster
+
+- Log in to the Equinix Metal [console](https://console.equinix.com/) and get
+the project API key from the project settings. If it doesn't exist create it.
+
+```sh
+export PACKET_API_KEY="<PROJECT_API_KEY>"
+```
+
+- Install the CAPI controllers using [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl).
+
+```sh
+clusterctl init --infrastructure packet
+```
+
+- Set env vars with cluster config.
+
+```sh
+# Get the project ID from the project settings in the console
+export PROJECT_ID="<PROJECT_ID>"
+
+# Use Paris metro (Equinix region)
+export METRO="pa"
+
+# Use Ubuntu 22.04 with cGroup v2 for Kepler
+export NODE_OS="ubuntu_22_04"
+
+# The pod and service CIDRs for the new cluster
+export POD_CIDR="192.168.0.0/16"
+export SERVICE_CIDR="172.26.0.0/16"
+
+# Use node type with Intel CPUs for RAPL support
+export CONTROLPLANE_NODE_TYPE="m3.small.x86"
+export WORKER_NODE_TYPE="m3.small.x86"
+
+# SSH key to use for access to nodes
+export SSH_KEY="<YOUR_SSH_KEY>"
+```
+
+- Generate cluster manifests.
+
+```sh
+clusterctl generate cluster wg-green-reviews \
+  --kubernetes-version v1.28.2 \
+  --control-plane-machine-count=1 \
+  --worker-machine-count=1 \
+  > wg-green-reviews.yaml
+```
```

Comment:
Cluster will be named `wg-green-reviews` and I went with the latest release of Kubernetes.

Reply:
WDYT about also exporting the kubernetes version as an env var to surface it and make it easier to update in the future?

Reply:
Yes good idea! I've added that.